tests/rbd/delete_clones_with_io.py

```diff
@@ -0,0 +1,94 @@
+import time
+
+from krbd_io_handler import krbd_io_handler
+from rbd.exceptions import RbdBaseException
+
+from ceph.parallel import parallel
+from tests.rbd.rbd_utils import Rbd
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(**kw):
+    """Delete Clones while IO is running on parent.
+
+    Args:
+        kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+
+    Pre-requisites :
+    We need atleast one client node with ceph-common package,
+    conf and keyring files
+
+    Test cases covered -
+    1) CEPH-9225 - RBD-Kernel_Client: Delete all Clone at once
+
+    Test Case Flow
+    1. Create a pool, an Image, 10 snapshots, 10 clones.
+    2. generate IO on the Image, in parallel delete the clones.
+
+    """
+    log.info("Running test - delete clones with IOs")
+    config = kw["config"]
+    rbd = Rbd(**kw)
+
+    pool = config.get("pool_name", rbd.random_string())
+    image = config.get("image_name", rbd.random_string())
+    size = "10G"
+
+    try:
+
+        def create_clone(*args):
+            if not rbd.create_pool(poolname=pool):
```

Comment:
we can use initial_rbd_config method here

Reply:
Will implement once #2001 is merged.